### PR TITLE
Fix typos and possibly a bug?

### PR DESCRIPTION
**Pull request by BenSandeen**
Fix a typo in comments, fix some whitespace, and possibly fix a bug??

I haven't been able to get the tests running, so I'm sorry if this fails tests

This is an attempt to resolve this issue: https://github.com/timescale/timescaledb-toolkit/issues/508

### CODE REVIEW

**Comment by rtwalker** [comment]
Thanks for the PR!

> I haven't been able to get the tests running, so I'm sorry if this fails tests

I just kicked them off, so we'll find out shortly :)

Were you trying `cargo pgx test pg14`? It's good for us to know when things don't work for others.

**Comment by BenSandeen** [comment]
> Were you trying `cargo pgx test pg14`? It's good for us to know when things don't work for others.

I was attempting to run `cargo pgx test pg12`; I'd have preferred to test against PG 14, but `pgx`'s installation somehow only managed to get PG 12 up and running.  And which directory am I supposed to run the tests in?  I tried both the root directory and `extension`.

**Review by JLockerman — APPROVED** [review]
Yup, that looks like a bug in the bias correction code. I suspect it's a significant contributor to issue https://github.com/timescale/timescaledb-toolkit/issues/508

**Comment by BenSandeen** [comment]
> Yup, that looks like a bug in the bias correction code. I suspect it's a significant contributor to issue #508

I spent some more time thinking about this, and I'm less confident than I originally was that it'll completely resolve that issue (although I still don't fully understand this code :P ).  There are some tests that are still passing when run against this PR that assert that the `estimate_count()` method returns values with greater error than we'd expect.  Specifically, the following tests that use a value for `b` of 12 or greater that assert a relative error of greater than 10%:
- https://github.com/timescale/timescaledb-toolkit/blob/64ddef116dc1a8b235589b51d4e6a85792804392/crates/hyperloglogplusplus/src/dense.rs#L302-L309
- https://github.com/timescale/timescaledb-toolkit/blob/64ddef116dc1a8b235589b51d4e6a85792804392/crates/hyperloglogplusplus/src/dense.rs#L329-L336
- https://github.com/timescale/timescaledb-toolkit/blob/64ddef116dc1a8b235589b51d4e6a85792804392/crates/hyperloglogplusplus/src/dense.rs#L347-L354
- https://github.com/timescale/timescaledb-toolkit/blob/64ddef116dc1a8b235589b51d4e6a85792804392/crates/hyperloglogplusplus/src/dense.rs#L356-L363

File: crates/hyperloglogplusplus/src/dense.rs [hunk 1]
```diff
@@ -194,8 +194,8 @@ impl<'s> Storage<'s> {
                             (idx, right_delta)
                         } else {
                             let idx = *l;
-                            if *r > 0 {
-                                *r -= 1;
+                            if *l > 0 {
+                                *l -= 1;
```
[the remark this answers is not in the document]
Comment (WireBaron) [reply crates/hyperloglogplusplus/src/dense.rs:198]: No, this is definitely a bug.  Great catch!

**Review by WireBaron — APPROVED** [review]

**Pull request merged**